### DISCUSSION TITLE
Import from Cookn and Living Cookbook

### DISCUSSION THREAD

**Discussion by GourmetSaint** [mealie-recipes/mealie#3372] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Both very popular recipe managers. Living Cookbook now debunked but had very good xml-based schema. When LC stopped, DVO attracted many users across with conversion support. However, not happy with the product and many new recipes. Mealie looks amazing but need to migrate years of recipes.

### Please provide a concise description of the feature that would resolve your issue.

Bulk import from LC and Cookn (now v14).

### Please consider and list out some caveats or tradeoffs made in your design decision

Tried exporting as html and hosting on local website and using mealie bulk import. Not reliable.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
Please share sample exports.
That's no guarantee it'll get implemented, but it can't even be considered without knowing what the data looks like.

**Comment by GourmetSaint** [comment]
Cookn Cookbook export attached
[Kids Cookin'.zip](https://github.com/mealie-recipes/mealie/files/14750853/Kids.Cookin.zip)
I have a large backup zip file as well (~50Mb).

**Comment by GourmetSaint** [comment]
I found a workaround. I purchased a Paprika licence, imported into Paprika, exported again and migrated into Mealie.
